ext/gd/libgd/gd_avif.c

```diff
@@ -81,7 +81,7 @@ static int quality2Quantizer(int quality) {
 
 	float scaleFactor = (float) AVIF_QUANTIZER_WORST_QUALITY / (float) MAX_QUALITY;
 
-	return round(scaleFactor * (MAX_QUALITY - clampedQuality));
+	return (int) round(scaleFactor * (MAX_QUALITY - clampedQuality));
```

Comment:
I think the float calculation is unnecessary here. The following should be good enough:
```suggestion
	return clampedQuality * AVIF_QUANTIZER_WORST_QUALITY / MAX_QUALITY;
```
But changing this now would introduce a subtle BC break, so just casting to int appears to be sensible for now.